docs/operations/downgrade-ondat-2.7-to-2.6.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Downgrade Ondat from 2.7.0 to 2.6.0"
+linkTitle: Downgrade Ondat from 2.7.0 to 2.6.0
+weight: 150
+---
+
+# Overview
+
+This guide will walk you through how to downgrade from Ondat v2.7.0 to v2.6.0. You can use this procedure should you decide that you need to roll back after upgrading to 2.7.0.
+
+As part of the 2.7.0 release we are implementing a new design for mapping your Kubernetes volumes to the underlying data storage containers on disk. This will make operation and debugging of any issues much easier, as from 2.7.0 the storage files on disk will have the same name (UUID) as the Physical volumes (PV) in Kubernetes.
```

Comment:
I don't think this is correct. The PVC name != CP volume UUID != the DP blob files names. For example, here's my pvc `pvc-reptile`:
```
>>>>>>>> $ kubectl get pvc
NAME          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
pvc-reptile   Bound    pvc-aa74542d-3871-4188-a3bb-727a41244a1c   2Gi        RWO            storageos-rep   23h
```

But it's CP volume UUID is `bbbcf1d6-15eb-4d9a-9db2-a08ff740ed0a`:
```
>>>>>>>> $  kubectl exec -n storageos storageos-cli-65b99ccf4c-qvcrd -- storageos describe volume pvc-aa74542d-3871-4188-a3bb-727a41244a1c  | head -n2
ID                  bbbcf1d6-15eb-4d9a-9db2-a08ff740ed0a
Name                pvc-aa74542d-3871-4188-a3bb-727a41244a1c
```

And the blob files will be named something different entirely because the blob files are per *deployment*. The blob file names have the syntax <deployment_uuid>.<blob_id>.blob:
```
root@alexr-worker1:~# ll /var/lib/storageos/data/dev1/
total 367808
drwxr-xr-x 2 root root     4096 Apr  4 20:04 ./
drwxr-xr-x 4 root root     4096 Apr  4 20:04 ../
-rw------- 1 root root 54087680 Apr  5 13:09 deployment.c6dd3415-1522-4091-a087-3d72816c0f1b.0.blob
-rw------- 1 root root 54095872 Apr  5 13:09 deployment.c6dd3415-1522-4091-a087-3d72816c0f1b.1.blob
```

I would it at: "As part of the 2.7.0 release we are implementing a new design for mapping your Kubernetes volumes to the underlying data storage containers on disk."

Reply:
Updated to just say "As part of the 2.7.0 release we are implementing a new design for mapping your Kubernetes volumes to the underlying data storage containers on disk."